cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,24 @@
-describe('Sign Up page', () => {
-  it('should provide an ability to register new account', () => {
+import { generateUser } from '../support/generate';
 
+describe('User Registration and Login Tests', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io');
+  });
+
+  describe('Sign Up page', () => {
+    it('should provide an ability to register new account', () => {
+      cy.get(':nth-child(3) > .nav-link').click();
+      const user = generateUser();
```

Comment:
move it to 10nd line and add empty line between const and functions

Reply:
fix